backend/app.ts

```diff
@@ -1,56 +1,110 @@
 import express, { NextFunction, Request, Response } from "express";
 import path from "path";
-import cors from "cors";
-import passport from "passport";
-import { Strategy as LocalStrategy } from "passport-local";
-
-import { DBAuthenticationError } from "./error/500s";
-import { statusCodedError } from "./error/statusCodedError";
+const cors = require("cors");
 const multer = require("multer");
 const upload = multer();
+const passport = require("passport");
+const jwt = require("jsonwebtoken");
 
+const passportConfig = require("./passport");
 import { employeeRouter } from "./routes/employeeRouters";
+import { createOneEmployee } from "./controllers/employeeControllers";
+import { Employee } from ".prisma/client";
+import { EmployeeInsert } from "./interfaces/employeeInterface";
 
 // -------------------firing express app
 const app = express();
 
 //body paramter enable
 app.use(upload.array());
 
+passportConfig();
+
+app.use(passport.initialize());
 app.use(express.json());
 app.use(cors());
 app.use(express.urlencoded({ extended: false }));
 app.use(express.static(path.join(__dirname, "client/build")));
 
-passport.use(new LocalStrategy({ usernameField: "email" },
-  function (username, password, done) {
-    if (username !== "admin") {
-      return done(null, false, { message: "Username is wrong"});
-    }
-    if (password !== "password") {
-      return done(null, false, { message: "Password is wrong" });
-    }
-    return done(null, { username: "admin", message:"Login Successful" });
+// -------------------unprotected routes
+app.get("/unprotected", (request: Request, response: Response) => {
+  response.json({ msg: "unprotected" });
+});
```

Comment:
Is this needed?

Reply:
I want to keep it because it is the simplest way to demo the unprotected router.